dbcon/execplan/rewrites.cpp

```diff
@@ -59,12 +59,12 @@ SimpleFilter* castToSimpleFilter(execplan::TreeNode* node)
 bool commonContainsSemantic(const CommonContainer& common, execplan::ParseTree* node)
 {
   auto filter = castToFilter(node);
-  return filter && common.first.contains(node);
+  return filter && common.first.count(node) != 0;
```

Comment:
Instead of `count`, calling `find` might be more efficient here:
```c++
return filter && common.first.find(node) != common.first.end();
```

Reply:
std::set doesn't allow duplicates, so count is pretty the same, but shorter version

Reply:
```
      size_type
      count(const key_type& __x) const
      { return _M_t.find(__x) == _M_t.end() ? 0 : 1; }
```